docs/guides/entry_points.md

```diff
@@ -0,0 +1,90 @@
+
+There’s one concern though: while all visitors will be assigned in the experiment, only the visitors who see the recommendation carousel are exposed to a different experience. (We’ll ignore the impact of triggering an expensive computation for now.) Therefore, the two thirds of visitors who were assigned but never scrolled down the homepage should not be included in that experiment.
+
+For cases like that, we let you define an **Entry Point**: what event needs to happen for visitors to be exposed to a different experience, and considered enrolled to the experiment. It remains up to you to decide if this should be when the carousel enters the viewport, is fully or partially visible; it’s also up to your front-end developpers to trigger and log that event. But once that information is in your data warehouse, then you can use it to filter out which users participate in the experiment.
+
+::: note
+If you define an Entry point, all the time-limted metrics (“Conversion 7 days after assignment”) are based on the timestamp of the Entry point, not the assignment. 
+:::
+
+
+### Second example: delivery pricing
+
+Let’s imagine another example: an e-commerce sells large, unusual or even fragile items. Items might not all be in stock, depending on factors like size, colors or other aspect. If you sell plants, you might want to confirm it’s still alive. 
+
+Pricing for custom deliveries depends on a multitude of factors (destination, need for temperature control, tax status) and your logistical partner conveniently offers an API to send quotes. The issue is: the response time of that API is slow. Therefore, you trigger a request as soon as you visitors confirm the contents of their basket and the country where they want it delivered, but before you have confirmed that the items are in stock, for example. 
+
+You also can easily imagine offering renting a car, or a valuable asset and needing a quote from your insurance company, based on many factors: information about the person renting, time and duration, etc. 
+
+Let‘s say you want to test asking for fast and cheap options. Or you could be testing a new provider. Alternatively, you might want to test asking both for a quote and let customers compate. Whatever is the change you want to test, you will need to decide which API call to trigger before the customer is guaranteed to see pricing because you have to wait for confirmation, say, that the items are in stock.
+
+In this case too, the assignment happens when confirming basket contents and the address, but the entry point should be set to a little later, when the customer sees a quote.
+
+::: note
+
+If one of the API integration fails, and some customers assigned to Treatment don’t see the pricing quote because of a bug, then you’ll observe an [asymetric traffic](/statistics/sample-ratio-mismatch.md).
```

Comment:
The better option is the client’s existing monitoring of their API calls.  They should have an alert long before an SRM flags it as a problem. I’m trying to pepper some awareness of how SRM happens in a context where it could be relevant.
What I can do is mention that explicitly, so that people don‘t think they can rely on SRM detectors —because they shouldn’t.